exchanges/okx/okx_websocket.go

```diff
@@ -16,7 +16,6 @@ import (
 
```

Comment:
I will leave the following as comments so you can reply. These are the kinds of error messages I receive on `enableallpairs` for okx. I will leave them to you to decide OOS or not for now
```
[DEBUG] | WEBSOCKET | 01/12/2025 10:29:30 | Okx wss://ws.okx.com:8443/ws/v5/public: Message received: {"event":"error","msg":"Wrong URL or channel:candles,instId:BTC-USD-260626-80000-C doesn't exist. Please use the correct URL, channel and parameters referring to API document.","code":"60018","connId":"4bf01114"}
```

Reply:
Yeah I think it was intentionally routed by a separate business connection and subscription. I kept the TestBusinessWSCandleSubscriptions func just to demonstrate that. It will have to be correctly managed with a template update. But I will unresolve this for other peoples input if its coming up on general subscriptions.